src/components/grid/grid.ts

```diff
@@ -179,7 +178,7 @@ export class Grid extends Component<Props, SpreadsheetChildEnv> {
     },
     "CTRL+A": () => this.env.model.selection.loopSelection(),
     "CTRL+S": () => {
-      this.props.onSaveRequested?.();
```

Comment:
shouldn't `CTRL+S` be handled one layer higher ?

Reply:
As discussed, we remove the CTRL+S